serf/serf.go

```diff
@@ -1438,6 +1453,33 @@ func (s *Serf) resolveNodeConflict() {
 	}
 }
 
+//handlePrune is a simplified version of Reap() which only runs when the prune
```

Comment:
nit (space after slashes):
```suggestion
// handlePrune is a simplified version of Reap() which only runs when the prune
```